testsuite/features/support/constants.rb

```diff
@@ -135,11 +135,20 @@
 sle_base_channel =
   if ENV['PROVIDER'].include? 'podman'
```

Comment:
Let's use `$is_container_provider` here?

Reply:
Sure. We have the following in `env.rb` which is identical to the old code.

```ruby
$is_container_provider = ENV["PROVIDER"].include? 'podman'
```

Reply:
Yup, I was referring to this :)